handlers/eth1Account.go

```diff
@@ -361,16 +360,10 @@ func Eth1AddressWithdrawals(w http.ResponseWriter, r *http.Request) {
 	address := strings.Replace(vars["address"], "0x", "", -1)
 	address = strings.ToLower(address)
 
-	pageToken, err := strconv.ParseUint(q.Get("pageToken"), 10, 64)
+	withdrawals, nextPageToken, err := db.GetAddressWithdrawals(common.HexToAddress(address).Bytes(), 25, q.Get("pageToken"))
 	if err != nil {
-		logger.WithError(err).Errorf("error parsing page token")
+		logger.WithError(err).Errorf("error getting address withdrawals data")
 		http.Error(w, "Internal server error", http.StatusServiceUnavailable)
```

Comment:
Keep in mind that I now also return an ISE for reasons other than parsing errors.